server_addon/maya/server/settings/publishers.py

```diff
@@ -1371,6 +1600,52 @@ class PublishersModel(BaseSettingsModel):
     "ExtractModel": {
         "enabled": True,
         "optional": True,
-        "active": True,
+        "active": True
+    },
+    "ExtractAlembic": {
+        "enabled": True,
+        "families": [
+            "pointcache",
+            "model",
+            "vrayproxy.alembic"
+        ],
+        "attr": "",
+        "attrPrefix": "",
+        "autoSubd": False,
+        "bake_attributes": [],
+        "bake_attribute_prefixes": [],
+        "dataFormat": "ogawa",
+        "eulerFilter": False,
```

Comment:
This defaulted to True before, by making this `False` with this PR this now means there is a difference in default behavior.